### ISSUE TITLE
Is there a way to customisable add right click menu on nodes?

### ISSUE THREAD

**Issue by ganeshpaharan** [bumbeishvili/org-chart#404]
Hello,
Is there a way to add custom context menu such as “Open Details, Delete, Add Child etc” when right clicking on a node? 
Thanks in advance

**Comment by bumbeishvili** [comment]
Yes, you have the possibility to access `this` inside `chart.nodeUpdate` method, which is a DOM element and then you can do anything 


Here is a sample - https://stackblitz.com/edit/js-vhhxqs?file=index.html

**Comment by ganeshpaharan** [comment]
Thanks very much. Appreciate your quick response
This is very helpful for me. 
Cheers 🍻

**Comment by muammadibal** [comment]
> Yes, you have the possibility to access `this` inside `chart.nodeUpdate` method, which is a DOM element and then you can do anything
> 
> Here is a sample - https://stackblitz.com/edit/js-vhhxqs?file=index.html

is new d3.OrgChart() and d3.select from your component or from separate import d3js?
i had try it from link above but it doesn't work in react, do you have an example using react?
